src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5707,12 +5702,47 @@ public override IOperation VisitDefaultValue(IDefaultValueOperation operation, i
 
         public override IOperation VisitTypeParameterObjectCreation(ITypeParameterObjectCreationOperation operation, int? captureIdForResult)
         {
-            return new TypeParameterObjectCreationExpression(Visit(operation.Initializer), semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
+            return new TypeParameterObjectCreationExpression(initializer: null, // PROTOTYPE(dataflow): Dropping initializer for now to enable test hook verification
```

Comment:
If you have to fix up merge conflicts, consider reverting this change I have the TypeParameterObjectCreation pr ready to go as soon as tests pass on [https://github.com/dotnet/roslyn/pull/27308](https://github.com/dotnet/roslyn/pull/27308).

Reply:
> If you have to fix up merge conflicts, consider reverting this change I have the TypeParameterObjectCreation pr ready to go as soon as tests pass on [https://github.com/dotnet/roslyn/pull/27308](https://github.com/dotnet/roslyn/pull/27308).

I think the test hook will not be "green" without this change. We want to remain "green" all the time

---
In reply to: [192874642](https://github.com/dotnet/roslyn/pull/27393#discussion_r192874642) [](ancestors = 192874642)